### ISSUE TITLE
DNS protection setting is not getting saved permanently 

### ISSUE THREAD

**Issue by sam20e** [AdguardTeam/AdguardForiOS#1966]
Hello. I tried sending an email to support email 2 times now. It’s been more than a month without any reply that’s why openings a bug report.

I’m using iOS and latest adguard. I’m using adguards dns protection which successfully unblocks some of our countries blocked websites. But whenever I switch from 4g to wifi or vice versa the dns protection is automatically turned off.  This is not happening every single time but randomly happening.

I tried reinstalling the app also but it’s a little bit inconvenient to go back and turn on dns protection Everytime when there is a network change. Can you check and advise?

### Prerequisites

Please answer the following questions for yourself before submitting an issue. 

- [x] I am running the latest version
Yes
- [x] I checked the documentation and found no answer
Yes
- [x] I checked to make sure that this issue has not already been filed

Yes

### Issue Details

- **AdGuard version:**
Latest
- **Device model and storage size:**
iPhone 11 pro
- **Operating system and version:**
15.4
- **Browser or App:**
DNS service

### Expected Behavior

When I enable DNS protection, I'd expect it to be turned on permanently.

### Actual Behavior

DNS protection is being turned off automatically/randomly

### Screenshots

https://imgur.com/ZUsXUaH

**Comment by zzebrum** [comment]
@sam20e I'v very sorry for the late reply!

is Restart protection when network changes enabled?

**Comment by sam20e** [comment]
> @sam20e I'v very sorry for the late reply!
> 
> is Restart protection when network changes enabled?

Hi @zzebrum thanks for the reply. No that option is not enabled. Should I enable that? Even if I do that, will it help with DNS protection settings?

**Comment by AlKulagin23** [comment]
@sam20e Hi again!
Could you please send me your ticket ID or the email address you contacted us from?

**Comment by sam20e** [comment]
Hi @AlKulagin23 one ticket id is 581849 before that I sent another enquiry 2 months ago but i dont have the ticket id for that. After that I again sent another ticket 589984 (3 weeks ago) and until now no reply. 

I'm using adguard for very long and never had these kinda delay. I understand that some of your colleagues are from ukraine so I don't blame them but then again I feel like our tickets should be answered sooner or later.. 

I even opened a forum thread but didn;t get any proper reply.. so not sure what else to do thats why opened a bug case here. 

Thank you for your time.

**Comment by AlKulagin23** [comment]
Thanks for the clarification! I replied to you in ticket #589984.
